README.md

```diff
@@ -130,11 +130,11 @@ Python (3.9 or higher), and install python3 packages using `./scripts/pysync` (r
 ```sh
 # Create repository in .neon with proper paths to binaries and data
 # Later that would be responsibility of a package install script
-> ./target/debug/neon_local init
+> cargo neon init
```

Comment:
If I have built both debug and release versions, which one will `cargo neon` use?

Reply:
The debug version (cargo run without --release builds the debug version)

Reply:
Based on https://github.com/neondatabase/neon/blob/a5615bd8ea202b24d91d506cae230de5e25bdeb9/control_plane/src/local_env.rs#L289, neon_local will pick debug/release based on noen_local's path, so in this case, the debug version will be running. We can add a new alias like `neonr` for the release version.

Reply:
Got it, thanks!

Reply:
Note that (from my memories) the `neon_distrib_dir` is only picked up once when creating the `.neon/config` toml file, during what is now `cargo neon init`.

I think not changing this or even not adding a `neonr` is good, because most of the time you'll find that the `.neon` exists already and so it'll just work.

Didn't check, but I'd imagine `cargo neon --release init` would work, as a short hand for initialize with using the `release` dir.